demo/public/modules-demo/demo.js

```diff
@@ -1,5 +1,5 @@
 import { Engine as ShakaEngine } from "p2p-media-loader-shaka";
-import { Engine as HlsEngine } from "p2p-media-loader-hls";
+import { Engine as HlsEngine } from "p2p-media-loader-hlsjs";
```

Comment:
Here was the error. 
I imported here types from our workspace, but on demo page [in public folder] it takes import from our importmap. If there are no builded es modules, the page doesn't work properly.